### PR TITLE
Ava UI: Add Control+Cmd+F HotKey for Mac OS

### LINKED ISSUE #3981: Mac default Fullscreen key must be changed
### What's the issue you encountered?

The F11 Key that Riujinx uses for fullscreen is already used by the mac system for the "show desktop" action, and Riujinx wont be full screen.

### How can the issue be reproduced?

Just press Fn + F11 on Mac

### Environment?

- Ryujinx version: 1.1.0 (mac-1)</br>

- System Specs:
  - OS: *macOS Ventura (13.0.1)*

### PR DESCRIPTION
**Pull request by gnisman**
Most Apps on Mac OS use ^+⌘+F (Control+Cmd+F) to toggle fullscreen mode
Added additional HotKey to toggle full screen by ^+⌘+F. 
Tested on Mac OS Ventura
Closes #3981

### CODE REVIEW

File: Ryujinx.Ava/UI/Windows/MainWindow.axaml.cs [hunk 1]
```diff
@@ -329,6 +329,7 @@ public void LoadHotKeys()
         {
             HotKeyManager.SetHotKey(FullscreenHotKey,  new KeyGesture(Key.Enter, KeyModifiers.Alt));
             HotKeyManager.SetHotKey(FullscreenHotKey2, new KeyGesture(Key.F11));
+            HotKeyManager.SetHotKey(FullscreenHotKeyMacOS, new KeyGesture(Key.F, KeyModifiers.Control | KeyModifiers.Meta)); //Mac OS Control+Cmd+F
```
Comment (gdkchan) [inline Ryujinx.Ava/UI/Windows/MainWindow.axaml.cs:332]: The other entries should be aligned, or the alignment should be removed for consistency.

**Review by gdkchan — APPROVED** [review]
lgtm, thanks.

**Review by riperiperi — APPROVED** [review]

File: Ryujinx.Ava/UI/Windows/MainWindow.axaml.cs [hunk 2]
```diff
@@ -327,10 +327,11 @@ public static void UpdateGraphicsConfig()
 
         public void LoadHotKeys()
         {
-            HotKeyManager.SetHotKey(FullscreenHotKey,  new KeyGesture(Key.Enter, KeyModifiers.Alt));
-            HotKeyManager.SetHotKey(FullscreenHotKey2, new KeyGesture(Key.F11));
-            HotKeyManager.SetHotKey(DockToggleHotKey,  new KeyGesture(Key.F9));
-            HotKeyManager.SetHotKey(ExitHotKey,        new KeyGesture(Key.Escape));
+            HotKeyManager.SetHotKey(FullscreenHotKey,      new KeyGesture(Key.Enter, KeyModifiers.Alt));
+            HotKeyManager.SetHotKey(FullscreenHotKey2,     new KeyGesture(Key.F11));
+            HotKeyManager.SetHotKey(FullscreenHotKeyMacOS, new KeyGesture(Key.F, KeyModifiers.Control | KeyModifiers.Meta)); //Mac OS Control+Cmd+F
```
Comment (riperiperi) [inline Ryujinx.Ava/UI/Windows/MainWindow.axaml.cs:332]: I'm not sure if the comment is necessary.

Comment (AcK77) [reply Ryujinx.Ava/UI/Windows/MainWindow.axaml.cs:332]:
```suggestion
            HotKeyManager.SetHotKey(FullscreenHotKeyMacOS, new KeyGesture(Key.F, KeyModifiers.Control | KeyModifiers.Meta));
```
> I'm not sure if the comment is necessary.

Yeah you can remove this comment, it's not.

Comment (gnisman) [reply Ryujinx.Ava/UI/Windows/MainWindow.axaml.cs]: Done

**Pull request merged**